src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -290,13 +290,15 @@ private function detectMappingType(string $directory, ContainerBuilder $containe
         $glob = new GlobResource($directory, '*', true);
         $container->addResource($glob);
 
+        $quotedMappingObjectName = preg_quote($this->getMappingObjectDefaultName());
```

Comment:
```suggestion
        $quotedMappingObjectName = preg_quote($this->getMappingObjectDefaultName(), '/');
```


According to the docs:
`
If the optional delimiter is specified, it will also be escaped. This is useful for escaping the delimiter that is required by the PCRE functions. The / is the most commonly used delimiter. 
`